### DISCUSSION TITLE
Is it possible to use css functions like clamp() / min() / max()?

### DISCUSSION THREAD

**Discussion by Poolshark** [chakra-ui/chakra-ui#2661] [Q&A] [answered]
I was wondering if it is possible to use css functions like e.g. `clamp()`, `min()` and `max()` somehow? I tried with the `sx` property but no chance. Furthermore it seems that I cannot use `sx` within a theme - which is also strange.

I have also tried to add it directly to e.g. the css `font-size` property, but also here no success.

```
fontSize: 'clamp(0.8em, 2vw, 3em)'   <-- in theme
fontSize={'clamp(0.8em, 2vw, 3em)} <-- in component
sx={{fontSize: 'clamp(0.8em, 2vw, 3em)'}} <-- in component
```

**Comment by with-heart** [comment, marked as answer]
Not sure why this isn't working for you. Seems to be working fine here: https://codesandbox.io/s/exciting-jang-uon48?file=/src/App.tsx

> I tried with the sx property but no chance. Furthermore it seems that I cannot use sx within a theme

Theme objects use the same syntax as the objects passed to `sx`, so you don't need to use `sx` inside of your theme.

**Reply by Poolshark** [reply]
Thanks for the quick answer!

My bad! Is is working but unfortunately Safari 14.0.1 does not allow `clamp()` on the `font-size` property...

**Reply by with-heart** [reply]
@Poolshark That would do it! That's unfortunate.
